imagersite/imagersite/templates/imagersite/base.html

```diff
@@ -0,0 +1,54 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="eng">
+
+    <head>
+        <meta charset="utf-8">
+        <link href="{% static './static/style.css' %}" rel="stylesheet">
+        <title style='color: red'>{% block title %}imagersite homepage{% endblock %}</title>
+    </head>
+    
+    <body>
+    <div id="header">
+    {% block header %}
+        <h1 style='color: red'>imagersite homepage</h1>
+        <div class="img">
+        <image src="/MEDIA/user_images/animate0.png" width="600" height="250"/>
```

Comment:
If you want a default image you need to serve it through your static directory. nothing from MEDIA goes to github so I don't see this image.

The assignment also asks you to serve a random photo from the database, or if nothing exists a stock photo. You should add this functionality.